components/WorkoutForm.js

```diff
@@ -0,0 +1,262 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { exercises } from "@/lib/exercises";
+import findExerciseById from "@/utils/findExerciseById";
+
+// Styled Components
+const Header = styled.header`
+  text-align: center;
+  margin-bottom: 1rem;
+`;
+
+const WrapperForm = styled.div`
+  display: flex;
+  justify-content: center;
+  background-color: white;
+  border-radius: 10px;
+  padding: 1rem;
+  margin: 1rem;
+  padding-top: 0;
+  position: absolute;
+  top: 0;
+  z-index: 1;
+`;
+
+const FormContainer = styled.div`
+  width: 100%;
+  max-width: 800px;
+  padding: 1rem;
+  padding-top: 0;
+  margin: 0 auto;
+`;
+
+const Fieldset = styled.fieldset`
+  border: none;
+  margin-bottom: 1rem;
+  padding: 1rem;
+  border-radius: 8px;
+  box-shadow: 0 4px 8px #0000001a;
+`;
+
+const Legend = styled.legend`
+  font-weight: bold;
+  margin-bottom: 0.5rem;
+`;
+
+const Label = styled.label`
+  display: block;
+  margin-bottom: 1rem;
+  font-weight: bold;
+`;
+
+const Input = styled.input`
+  width: 100%;
+  padding: 0.5rem;
+  margin-top: 0.5rem;
+  border-radius: 4px;
+  border: 1px solid #ddd;
+`;
+
+const Select = styled.select`
+  width: 100%;
+  padding: 0.5rem;
+  margin-top: 0.5rem;
+  border-radius: 4px;
+  border: 1px solid #ddd;
+`;
+
+const AddButton = styled.button`
+  background-color: #3498db;
+  color: #ffffff;
+  border: none;
+  padding: 0.75rem 1.5rem;
+  border-radius: 4px;
+  cursor: pointer;
```

Comment:
a coach told us before that a button element automatically is set as "cursor: pointer" so this line shouldn't be necessary. Didn't work out in our case though 😅